frontend/src/components/annotation/sidebar_factory.vue

```diff
@@ -37,15 +37,33 @@
     :schema_id="annotation_ui_context.label_schema.id"
     :loading="annotation_ui_context.get_current_ann_ctx() ? annotation_ui_context.get_current_ann_ctx().rendering :  true"
     :current_instance="annotation_ui_context.get_current_ann_ctx() && annotation_ui_context.get_current_ann_ctx().current_instance"
-    @on_select_instance="(e) => $emit('on_select_text_instance', e)"
-    @delete_instance="(e) => $emit('delete_text_instance', e)"
+    @on_select_instance="(e) => $emit('on_select_instance', e)"
```

Comment:
Changed `on_select_text_instance` to `on_select_instance` because these events are not only used for processing text annotations. Then did the same thing with other events.